### PR TITLE
fix typos for Get Activity Version SDK calls

### PR DESCRIPTION
**Pull request by zhuliice**
fix the issue: https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/issues/30

### CODE REVIEW

**Review by szilvaa — APPROVED** [review]

**Comment by zhuliice** [comment]
@szilvaa see @augustogoncalves 's [comment here](https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/issues/30#issuecomment-1032080202), I think I need to work more on this PR..

File: src/Autodesk.Forge.DesignAutomation/ApiClient.gen.cs [hunk 1]
```diff
@@ -169,11 +170,25 @@ public async System.Threading.Tasks.Task DeleteActivityVersionAsync (string id,
...
         /// <param name="version">Version to retrieve (integer).</param>
         /// <returns>Task of Activity</returns>
+        [Obsolete("This method is obsolete. Call GetActivityVersionAsync instead.", false)]
         public async System.Threading.Tasks.Task<Activity> GeActivityVersionAsync (string id, int version)
         {
-             var response = await this.ActivitiesApi.GeActivityVersionAsync(id, version);
-             return response.Content;
-
+             return await this.GetActivityVersionAsync(id, version);
+        }
```
Comment (zhuliice) [inline src/Autodesk.Forge.DesignAutomation/ApiClient.gen.cs:177]: The old one usage generates a compiler warning

File: src/Autodesk.Forge.DesignAutomation/ApiClient.gen.cs [hunk 2]
```diff
@@ -169,11 +170,25 @@ public async System.Threading.Tasks.Task DeleteActivityVersionAsync (string id,
...
+        /// Gets the details of the specified version of the Activity.
+        /// </remarks>
+        /// <exception cref="HttpRequestException">Thrown when fails to make API call</exception>
+        /// <param name="id">Name of Activity (unqualified).</param>
+        /// <param name="version">Version to retrieve (integer).</param>
+        /// <returns>Task of Activity</returns>
+        public async System.Threading.Tasks.Task<Activity> GetActivityVersionAsync(string id, int version)
+        {
+            var response = await this.ActivitiesApi.GetActivityVersionAsync(id, version);
+            return response.Content;
```
Comment (zhuliice) [inline src/Autodesk.Forge.DesignAutomation/ApiClient.gen.cs:191]: the new one has correct name

File: src/Autodesk.Forge.DesignAutomation/Http/ActivitiesApi.gen.cs [hunk 3]
```diff
@@ -582,7 +594,22 @@ public async System.Threading.Tasks.Task<HttpResponseMessage> DeleteActivityVers
         /// <param name="id">Name of Activity (unqualified).</param>/// <param name="version">Version to retrieve (integer).</param>
         /// <returns>Task of ApiResponse<Activity></returns>
         
-        public async System.Threading.Tasks.Task<ApiResponse<Activity>> GeActivityVersionAsync (string id, int version, string scopes = null, IDictionary<string, string> headers = null, bool throwOnError = true)
+        [Obsolete("This method is obsolete. Call GetActivityVersionAsync instead.", false)]
+        public async System.Threading.Tasks.Task<ApiResponse<Activity>> GeActivityVersionAsync(string id, int version, string scopes = null, IDictionary<string, string> headers = null, bool throwOnError = true)
+        {
+            return await this.GetActivityVersionAsync(id, version, scopes, headers, throwOnError);
+        }
```
Comment (zhuliice) [inline src/Autodesk.Forge.DesignAutomation/Http/ActivitiesApi.gen.cs:601]: the same, the old one usage generates a compiler warning

**Pull request merged**